apigee_api_catalog.install

```diff
@@ -134,3 +134,10 @@ function apigee_api_catalog_update_8806() {
 function apigee_api_catalog_update_8807() {
   return \Drupal::service('apigee_api_catalog.updates')->update8807();
 }
+
+/**
+ * Removed yml extension from field_apidoc_file_link and field_apidoc_spec allowed values.
+ */
+function apigee_api_catalog_update_8808() {
```

Comment:
The PR should merge in the 3.x branch as well.

Reply:
The update can be 8810

Reply:
@phdhiren 
3.x branch https://github.com/apigee/apigee-api-catalog-drupal/pull/176